DotNet/overrides.js

```diff
@@ -54,4 +54,9 @@ module.exports = {
 
     "should use regions padding": {skipEmit: true},
     "should use regions padding with vg": {skipEmit: true},
+
+    'regionId can be specified as part of checkSettings classic': {skipEmit: true},
+    'regionId can be specified as part of checkSettings with vg': {skipEmit: true},
+    'coded regions with selectors or elements should automatically include a regionId classic': {skipEmit: true},
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=9690162)
```suggestion
    "coded regions with selectors or elements should automatically include a regionId classic": {skipEmit: true},
```

Reply:
@Yarden-Ingber This seems a bit abrupt. Can you communicate changes like this are coming rather than simply turning them on? And who was involved in setting the coding styles? Not to mention that this repo is getting merged to the JS monorepo soon, where we handle coding style on our own w/ linting rules and pre-push checks. What happens if the rules you've set are different than what we've set in our repo?